packages/design-tokens/package.json

```diff
@@ -30,11 +30,12 @@
     "@lmc-eu/commitlint-config": "1.0.10",
     "@lmc-eu/conventional-changelog-lmc-github": "2.0.0",
     "@lmc-eu/prettier-config": "1.2.2",
-    "@lmc-eu/stylelint-config": "3.0.0",
+    "@lmc-eu/stylelint-config": "5.0.0",
```

Comment:
Wow, upgraded by two versions and nothing broke? 👀 🎩

Reply:
Life is like a box of chocolates. You never know what you're gonna get.